.github/workflows/create-release-branch.yml

```diff
@@ -60,7 +60,9 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 'lts/*' # Always use Node LTS for building dependencies.
-      - run: yarn
```

Comment:
Not sure why we use yarn here instead of using --frozen-lockfile. Let me know if this was intentional.

Reply:
Using `--frozen-lockfile` throughout probably isn't the worst idea.